theano/gof/op.py

```diff
@@ -630,9 +634,14 @@ def __call__(self, *inputs, **kwargs):
                             (i, ins, node), stacklevel=2)
                         run_perform = False
                     elif config.compute_test_value == 'raise':
+                        sio = StringIO()
+                        for subtr in getattr(ins.tag, "trace", []):
+                            traceback.print_list(subtr, sio)
+                        detailed_err_msg = sio.getvalue()
+
                         raise ValueError(
-                            'Cannot compute test value: input %i (%s) of Op %s missing default value' %
-                            (i, ins, node))
+                            'Cannot compute test value: input %i (%s) of Op %s missing default value. %s' %
```

Comment:
idem.
In fact add somethine line:

default value. %s'  -> default value. \nStacktrace when the variable was created: %s'